py-polars/Makefile

```diff
@@ -25,11 +25,11 @@ fmt: venv  ## Run autoformatting and linting
 	$(VENV_BIN)/isort .
 	$(VENV_BIN)/black .
 	$(VENV_BIN)/blackdoc .
-	$(VENV_BIN)/pyupgrade --py37-plus `find polars/ tests/ -name "*.py" -type f`
+	$(VENV_BIN)/pyupgrade --py37-plus `find polars docs tests tests_parametric -name "*.py" -type f`
 	cargo fmt --all
 	-dprint fmt
 	-$(VENV_BIN)/mypy
-	-$(VENV_BIN)/flake8 polars tests docs
+	-$(VENV_BIN)/flake8
```

Comment:
I included those targets to account for people creating a non-standard virtual environment folder in their workspace.

Reply:
Is covered by the config in the file (where it should be): https://github.com/pola-rs/polars/blob/master/py-polars/.flake8#L16